build/package/LICENSE

```diff
@@ -224,24 +224,6 @@ conditions of the following licenses.
     This project is free software released under the MIT license:
     http://www.opensource.org/licenses/mit-license.php
 
-- 'buffers' in extension/dist/ext/extension.js
```

Comment:
When I run `yarn licenses list --production`, I still see many of these licenses that have been removed. I assume I'm doing something wrong. Is there a different way to generate the right list?

Reply:
Yeah you are right. So I was doing `yarn list --production` but thought only the highlighted items were production packages. I will get this fixed

Reply:
Actually, it looks like this might be an issue with the yarn licenses command: [yarnpkg/yarn#2567)](https://togithub.com/yarnpkg/yarn/issues/2567)

That bug is marked as closed, but there are reports that it's still broken.

If you used `yarn list --production`, that's probably the right list.

Reply:
Yeah so what I did was use `yarn list --production` and only grabbed the main dependencies, and their transitive dependencies, that were listed in the `package.json` file's `dependencies` section. Any main/transitive deps listed in the `package.json` file's `devDependencies` section were ignored. If this is what is wanted I would say the current listed items would be correct.

As I ran `yarn licensees list --production` I am seeing items listed in the `package.json` as `devDependencies` being listed.

Reply:
Sorry for the delay in looking at this. I ran `yarn list --production` and there's a large list of dependencies that aren't listed in `build/package/{LICENSE,NOTICE,NONOTICE}` in this PR. Maybe something is still wrong with the command I'm running or my environment? Or maybe these are excluded for a reason?

```
@cspotcode/source-map-support@0.8.1
@grpc/proto-loader@0.7.4
@jridgewell/resolve-uri@3.1.0
@jridgewell/sourcemap-codec@1.4.14
@jridgewell/trace-mapping@0.3.9
@protobufjs/aspromise@1.1.2
@protobufjs/base64@1.1.2
@protobufjs/codegen@2.0.4
@protobufjs/eventemitter@1.1.0
@protobufjs/fetch@1.1.0
@protobufjs/float@1.0.2
@protobufjs/inquire@1.1.0
@protobufjs/path@1.1.2
@protobufjs/pool@1.1.0
@protobufjs/utf8@1.1.0
@tsconfig/node10@1.0.9
@tsconfig/node12@1.0.11
@tsconfig/node14@1.0.3
@tsconfig/node16@1.0.3
@types/long@4.0.2
acorn-walk@8.2.0
acorn@8.8.2
ansi-regex@5.0.1
ansi-styles@4.3.0
arg@4.1.3
buffers@0.1.1
chainsaw@0.1.0
cliui@7.0.4
color-convert@2.0.1
color-name@1.1.4
create-require@1.1.1
diff@4.0.2
emoji-regex@8.0.0
escalade@3.1.1
get-caller-file@2.0.5
has-flag@4.0.0
is-fullwidth-code-point@3.0.0
lodash.camelcase@4.3.0
long@4.0.0
make-error@1.3.6
minimist@1.2.7
ms@2.1.2
omega-edit@0.9.34
os-paths@7.3.0
protobufjs@7.2.0
require-directory@2.1.1
string-width@4.2.3
strip-ansi@6.0.1
supports-color@7.2.0
traverse@0.3.9
v8-compile-cache-lib@3.0.1
wrap-ansi@7.0.0
y18n@5.0.8
yargs-parser@20.2.9
yargs@16.2.0
yn@3.1.1
```

Reply:
Yeah looking into more maybe these do get included into the packages based on the differences between `yarn list --prod` and `yarn list --dev`. I will get these added

Reply:
@stevedlawrence I believe this should be good now. `omega-edit` was listed in the `build/package/NOTICE` file. Items:

```bash
@grpc/proto-loader
long
```

Were added to the `build/package/NONOTICE` file. The rest got added to `build/package/LICENSE`.